### PR TITLE
Add initial contributing section

### PR DESCRIPTION
**Pull request by dholbach**
This adds a first proposing changes guide

Addresses: #237

Authored-by: Alison Dowdney <alison@weave.works>
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
@alisondy I took the contents from hackmd, fixed tiny small things like `markdownlint` and added some punctuation. I think it'd be good to start landing this work - this guide was already looking great.

**Comment by stefanprodan** [comment]
I don’t think a Contributing section should be only about docs, but about flux2 and the gitops toolkit controllers.

**Comment by dholbach** [comment]
Ok - that makes sense. Maybe we import our current contributing doc and plug it in the right place and create a general structure for all the Flux things.

File: content/en/docs/contributing/docs/proposing-changes.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+
+#### Addressing feedback locally
+
+1. After making your changes, amend your previous commit:
+
+   ```bash
+   git commit -as --amend
+   ```
+
+   - `-a`: commits all changes
```
Comment (alisondy) [inline content/en/docs/contributing/docs/proposing-changes.md:203]:
```suggestion
   - `-as`: commits all changes and adds your signoff
```

**Review by alisondy — CHANGES_REQUESTED** [review]
looks good just one minor suggestion

**Comment by dholbach** [comment]
https://deploy-preview-262--fluxcd.netlify.app/docs/contributing/ as a simple starting point.

File: hack/import-flux2-assets.sh [hunk 2]
```diff
@@ -168,5 +168,22 @@ gen_crd_doc() {
     mkdir static
   fi
   curl -s -# -Lf https://raw.githubusercontent.com/fluxcd/flux2/main/install/flux.sh -o static/install.sh
+}
+
+{
+  # Not very nice - we should re-write `hack/gen-content.sh` in python and
+  # - make it read csv
+  # - (optionally re-write the title)
```
Comment (dholbach) [inline hack/import-flux2-assets.sh:176]: I filed https://github.com/fluxcd/website/issues/263 for this.

**Review by alisondy — APPROVED** [review]

**Pull request merged**